samples/DataMockerSample/UITests/DataMockerSample.UITests.csproj

```diff
@@ -26,18 +27,22 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="Newtonsoft.Json">
-      <HintPath>..\packages\Newtonsoft.Json.9.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="DataMocker.UITest, Version=1.0.7982.18008, Culture=neutral, PublicKeyToken=null">

```

Comment:
could you please get rid of packages and switch over to  PackageReference instead? Ideally for all packages